docs/design/field-references.md

```diff
@@ -42,27 +49,30 @@ spec:
           name: sleeper2
         spec:
           sleepFor: 4
-          wakeupMessage: "{{sleeper1#status.message}}"
+          wakeupMessage: "!{sleeper1-status-message}"
   - name: sleeper3
-    dependsOn:
-    - sleeper2
+    references:
+    - name: sleeper2-spec
+      resource: sleeper2
+      path: spec
     spec:
       object:
         apiVersion: crd.atlassian.com/v1
         kind: Sleeper
         metadata:
           name: sleeper3
-        spec: "{{sleeper2#spec}}"
+        spec: "!{sleeper2-spec}"
 ```
 
 ## Referring to ServiceBinding outputs
 
-When Service Catalog processes a ServiceBinding, the output is placed in a Secret
+When Service Catalog processes a `ServiceBinding`, the output is placed in a `Secret`
 (since they might be secret). If they're not secret, it's convenient to directly
```

Comment:
"they may contain sensitive data"

Reply:
This text is originally from James, so I'm confident it is grammatically correct 😛

Reply:
Don't trust me! I am super-prone to making mistakes. I type slightly more quickly than I think in most cases.

But I think Nail here is not suggesting a grammatical change, but one which avoids the confusion of Secret vs secret, which I probably wrote just because it amused me. His version is better!